.github/workflows/dockerization.yml

```diff
@@ -28,7 +23,75 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install .
-    - name: Dockerization test
+
+    - name: Build base image
+      run: |
+        docker build --progress=plain -t openfl -f openfl-docker/Dockerfile.base .
+
+    - name: Create workspace image
+      run: |
+        fx workspace create --prefix example_workspace --template keras_cnn_mnist
+        cd example_workspace
+        fx plan initialize -a localhost
+        fx workspace dockerize --base_image openfl
+
+    - name: Create certificate authority for workspace
+      run: |
+        cd example_workspace
+        fx workspace certify
+
+    - name: Create signed cert for collaborator
+      run: |
+        cd example_workspace
+        fx collaborator create -d 1 -n charlie --silent
+        fx collaborator generate-cert-request -n charlie --silent
+        fx collaborator certify --request-pkg col_charlie_to_agg_cert_request.zip --silent
```

Comment:
I would advocate for running 2 collaborator containers

Reply:
The (now-deprecated) python test was based on a single collaborator. I will limit the scope of this PR as fixing the CI test.
But this is a fair concern. I have added a `TODO` in the action file to be addressed in later PRs.